builtin/credential/approle/path_role.go

```diff
@@ -108,6 +108,12 @@ type roleIDStorageEntry struct {
 func rolePaths(b *backend) []*framework.Path {
 	defTokenFields := tokenutil.TokenFields()
 
+	responseOK := map[int][]framework.Response{
+		http.StatusOK: {{
+			Description: "OK",
+		}},
+	}
```

Comment:
looks like this is already defined (I think?)
https://pkg.go.dev/github.com/hashicorp/vault/sdk@v0.6.1/framework#pkg-variables

Reply:
Unfortunately this struct is not quite the same. I was thinking of where to extract this to but haven't found a good spot yet, so keeping it here for now.